aip/general/0203.md

```diff
@@ -171,6 +163,15 @@ A resource with an unordered list **may** return the list in a stable order, or
 
 ## Rationale
 
+### Required set of annotations
+
+A field used in a request message must be either an input or an output.
```

Comment:
What does this mean?

Reply:
This section explains why some combination of annotations are required, and was trying to explain that a field is either and input or and output, and therefore a full description of the behavior should include at least one of required, optional, or output_only.